### ISSUE TITLE
[ES] WCAG Overview link

### ISSUE THREAD

**Issue by cmunch** [w3c/wai-std-gl-overview#24]
@slhenry @yatil 

Issue found in the Spanish version of this document, available at:
https://www.w3.org/WAI/standards-guidelines/es

When clicking in the "Resumen de WCAG" link (WCAG Overview), the user is getting a 404 error.

User should be directed to:
https://www.w3.org/WAI/standards-guidelines/wcag/

Instead of:
https://www.w3.org/standards-guidelines/wcag/

**Comment by yatil** [comment]
I fixed the link in https://github.com/w3c/wai-std-gl-overview/commit/79d8a610efc72cc4e476a3f8d9cf81dfc8e24c4f and have uploaded the change. 

¡Gracias!
